instructor/templates/instructor/assignment_sidebar.html

```diff
@@ -147,6 +147,16 @@
                 Sample Prep
             </a>
         </div>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+            {% if page_name == "facs_analyze" %}scb_ab_s_sidebar_page_name_selected{% endif %}'>
+            <a class="scb_s_assignment_sidebar_link {% if last_page_number < 12 %} disabled {% endif %}"
```

Comment:
Giovanni, I think I have answered your question, why this particular number was chosen, without any assumptions on background knowledge. If your question was regarding the functionality of the code segment itself, please see this PR #523 https://github.com/starteam/starcellbio_html/pull/523